packages/neutrino-preset-jest/src/index.js

```diff
@@ -6,13 +6,25 @@ const { tmpdir } = require('os');
 const clone = require('lodash.clonedeep');
 const loaderMerge = require('neutrino-middleware-loader-merge');
 
+function getFinalPath(path) {
+  if (path[0] === '/') {
```

Comment:
It looks like this is trying to determine if a path is relative or absolute. Is that the case? Could you explain what this is supposed to achieve?